phlop/app/nvidia/ncu.py

```
@@ -0,0 +1,57 @@
+# Nsight Compute CLI
+
+# https://docs.nvidia.com/nsight-compute/ProfilingGuide/index.html
+
+## samples
+#  ncu --metrics all
+#  ncu --metrics l1tex__data_bank_conflicts_pipe_lsu_mem_shared_op_st.sum
+#
+#
+
+import os
```

Ruff (0.8.0)

`os` imported but unused

Remove unused import: `os`

(F401)